ssv/partial_sig_container.go

```diff
@@ -1,39 +1,25 @@
 package ssv
 
 import (
-	"encoding/hex"
+	"bytes"
```

Comment:
why are all those changes (partial sig container) relevant to this PR?

Reply:
https://github.com/bloxapp/ssv-spec/commit/91ba801c3fd2c1734800eefa980e10e2055621b9

You added it here but I agree. Were you working on something else, or you want to keep it here?

Reply:
The previous PartialSigMessageContainer can't return the full PartialSigMessage which is a requirement for putting the preconsensus justification in the CD.
I initially thought you did this as part of the spec-rethink but now I think you did this for this PR and I propose we leave those changes